integration-cli/docker_cli_run_unix_test.go

```diff
@@ -242,13 +242,6 @@ func (s *DockerSuite) TestRunEchoStdoutWithCPUSharesAndMemoryLimit(c *check.C) {
 	}
 }
 
-func (s *DockerSuite) TestRunWithCpuset(c *check.C) {
```

Comment:
the single dash option wasn't even in the code https://github.com/docker/docker/pull/17724/files#diff-ae476143d40e21ac0918630f7365ed3cL92